.github/workflows/mas_check.yml

```diff
@@ -65,13 +65,10 @@ jobs:
       - name: rpy lint
         run: |
           cd renpy
-          ./renpy.sh "../mas0105/" lint &> renpy_output
-          python ../tools/renpy_lint_parser.py
-          cat renpy_output_clean
+          ./renpy.sh "../mas0105/" lint | grep -E -v "^$|Could not find image \(monika [0-9][^[:space:]]+ corresponding to attributes on say statement\.|'monika [0-9][^[:space:]]' is not an image\.|The image named 'monika [0-9][^[:space:]]+ was not declared\."
```

Comment:
This regex is covers too many cases. The previous one was very specific in catching forms of `monika #ccc+` (where `c` is any sprite code), `#c`, and specific other numbers/images. This is so an actual invalid code like `monika 1ef` would still appear in the log.

Reply:
We're only hiding renpy lint reports, won't our sprite checker messages still pass through?

Reply:
Yeah our spritechecker messages should still be pushed through since this only checks for renpy lint output

Reply:
that appears to be correct